clients/instance/ibm-pi-network-address-group.go

```diff
@@ -103,7 +103,7 @@ func (f *IBMPINetworkAddressGroupClient) AddMember(id string, body *models.Netwo
 }
 
 // Delete the member from a Network Address Group
-func (f *IBMPINetworkAddressGroupClient) DeleteMember(id, memberId string) (*models.NetworkAddressGroup, error) {
+func (f *IBMPINetworkAddressGroupClient) DeleteMember(id, memberId string) (models.Object, error) {
```

Comment:
Should this function still be returning anything if the return is models.Objects? Shouldn't this be modeled like a delete?

Reply:
Fixed in latest commit.